src/Robo/Task/Discovery/Alternatives.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Usher\Robo\Task\Discovery;
+
+use Robo\Task\BaseTask;
+use Robo\Result;
+use Robo\ResultData;
+
+/**
+ * Use this to figure out which binary to use from a series of alternatives.
+ *
+ * ``` php
+ * <?php
+ * // Figure out whether you should use more or less or cat on a system. The
+ * // result data will always contain a 'path' property, even if no alternative
+ * // could be resolved and the path is empty.
+ * $result = $this->task(Alternatives::class, 'more', 'less,cat')->run();
```

Comment:
```suggestion
 * $result = $this->task(Alternatives::class, 'more', ['less', 'cat'])->run();
```
I don't see the use of `explode` so I would expect an array, but perhaps I'm just not following this fully?